tests/api/core/content-manager/content-manager/history/history.test.api.ts

```diff
@@ -177,7 +178,7 @@ describeOnCondition(edition === 'EE')('History API', () => {
           description: 'Hello',
         },
       }),
-      updateEntry({
+      await updateEntry({
```

Comment:
await doesn't seem needed

Reply:
good catch I'll remove it in https://github.com/strapi/strapi/pull/20179